dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -515,4 +519,12 @@ public int compareTo(@NotNull Entry that) {
     public void setDefaultPaths(Map<DescriptorLanguage.FileType, String> defaultPaths) {
         this.defaultPaths = defaultPaths;
     }
+
+    public String getNextVersionName() {
```

Comment:
Let's get a test in place to make sure this value can't be edited by the user using `@PUT` on a workflow.